uint8 channel
---
bool result
